packages/runner-ct/src/app/RunnerCt.scss

```diff
@@ -32,6 +32,11 @@ body, html {
    background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI1IiBoZWlnaHQ9IjUiPgo8cmVjdCB3aWR0aD0iNSIgaGVpZ2h0PSI1IiBmaWxsPSIjZGJkYmRiIj48L3JlY3Q+CjxwYXRoIGQ9Ik0wIDVMNSAwWk02IDRMNCA2Wk0tMSAxTDEgLTFaIiBzdHJva2U9IiNjNGM0YzQiIHN0cm9rZS13aWR0aD0iMSI+PC9wYXRoPgo8L3N2Zz4=");
 }
 
+.aut-iframe-screenshotting {
+  height: min(100vh, 100%) !important;
+  overflow: scroll !important;
+}
+
```

Comment:
What styles is this conflicting with? I particularly don't understand the `min(100vh, 100%)`. Is the 100% the intrinsic content size, not the iframe parent's size?

Reply:
I figured this out just by creating lots of use cases and observing the output. Here's the use case this solves. Consider this example:

![image](https://user-images.githubusercontent.com/19196536/118740257-bcf56e00-b88e-11eb-922d-2a96bdafbb2b.png)

Without this CSS, the screenshot is like this. Note the default behvior of `cy.screenshot`, which is `cy.screenshot({ capture: 'fullPage' })` (as opposed to `cy.screenshot({ capture: 'viewport' })`.

![image](https://user-images.githubusercontent.com/19196536/118740292-d696b580-b88e-11eb-90d0-06517f319761.png)

This is the desired output - scroll down the page, stitch together the screenshot:

![image](https://user-images.githubusercontent.com/19196536/118740314-e57d6800-b88e-11eb-9826-efedf39c0c55.png).

We need `overflow: scroll` to be applied. If `overflow: scroll` is not applied, Cypress tries to scroll, take a screenshot, and stitch them together. That's why you end up with that weird duplicating behavior in the first image above.

`scroll: overflow` is only applied if you have a static height set - `height: 100%` won't work. By doing `min(100%, 100vh)` it will apply `height 100vh` to components with a height greater than the viewport (100%). This supports the desired behavior.

I have some ideas on a much less hacky way to do this moving forward, I will send you a message.

Reply:
Thank you for this detailed explanation.